.spelling

```diff
@@ -828,7 +836,6 @@ CodeFormatter
 StyleCop
 SytzeAndr
 yashrajbharti
- - CHANGELOG.md
```

Comment:
Why did you get rid of the per document sections?

Reply:
@ghvanderweg

Reply:
The per document sections were not being processed correctly, causing false negatives. I don't know if this is a regression in mdspell or an error in the testscript.

Reply:
cc @adityapatwardhan